tests/conftest.py

```diff
@@ -0,0 +1 @@
+pytest_plugins = ["pytester"]
```

Comment:
Today I learned about [pytester](https://docs.pytest.org/en/6.2.x/reference.html#pytester). Neat. This will require pytest>=6.2.